app/components/MessageThread.tsx

```
@@ -0,0 +1,94 @@
+                  [styles.messageSelf]: m.sender._id === user?._id,
+                })}
+                key={m._id}
+              >
+                <li>
+                  <article>
+                    <UserAvatar user={m.sender} size={34} />
+                    <div className={styles.messageBubble}>
+                      <header>{m.sender.name}</header>
+                      <div
+                        className={styles.messageContent}
+                        dangerouslySetInnerHTML={{ __html: m.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)